### PR TITLE
Scheduler with minikube

### PR DESCRIPTION
**Pull request by dimberman**
Make sure you have checked _all_ steps below.

### JIRA
- [ ] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-XXX

### Description
- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

- [ ] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -77,8 +77,8 @@ matrix:
     - python: "3.5"
       env: TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.9.0
   allow_failures:
-    - env: TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.8.0
     - env: TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.9.0
+    - env: TOX_ENV=py27-backend_postgres KUBERNETES_VERSION=v1.10.0
```
Comment (jzucker2) [inline .travis.yml]: Did you mean to change this to 1.1?

Also, you can specify [which version of kubernetes when starting minikube](https://kubernetes.io/docs/getting-started-guides/minikube/#specifying-the-kubernetes-version)

**Comment by dimberman** [comment]
@jzucker2 Here's a PR from @kimoonkim for minikube testing in travis https://github.com/apache-spark-on-k8s/kubernetes-HDFS/pull/34#pullrequestreview-110607678. I'm gonna review this PR today to see if we can't get this working but if you find time feel free to use this as a resource.

**Comment by dimberman** [comment]
This file in particular: https://github.com/kimoonkim/kubernetes-HDFS/blob/dcededb43463ad280db4d3e10d225b26de620ff5/tests/setup.sh

**Comment by dimberman** [comment]
Also this travis article: https://blog.travis-ci.com/2017-10-26-running-kubernetes-on-travis-ci-with-minikube

**Pull request merged**